spec/support/mock_parser.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'ostruct'
```

Comment:
The change isn't directly related to the fix, but it seems to be causing some tests to fail.

see: https://github.com/ruby-grape/grape-swagger/actions/runs/8771833802/job/24070096127?pr=924